src/main/java/io/jenkins/plugins/opentelemetry/job/runhandler/DefaultRunHandler.java

```diff
@@ -25,28 +24,31 @@
  * Use same root span name for all pull change request pipelines (pull request, merge request)
  * Use different span names for different branches.
  */
-@Extension(dynamicLoadable = YesNoMaybe.YES)
-@Symbol("basicSpanNamingStrategy")
-public class SpanNamingStrategy {
+@Extension
+public class DefaultRunHandler implements RunHandler {
```

Comment:
Is there any use case that someone could use this class?
Just in case this change could be a breaking change.

Reply:
I'm not worried, it was not even an interface with an SPI mechanism.
During the 4 years I have created and maintained the Jenkins Pipeline Maven Plugin, I have not seen anyone use the extension points I introduced :-)
We will document it in the release notes.